src/Components/Header/Header.js

```diff
@@ -19,26 +31,49 @@ export const Header = () => {
 					htmlFor='input-search'
 				/>
 
-				<input
-					id='input-search'
-					className='input-search'
-					placeholder='Search for videos...'
-				/>
-
-				<FeSearch
-					className='search-icon'
-					stroke='black'
-					width='2rem'
-					height='1.5rem'
-				/>
+				<div className='p-relative'>
+					<input
+						id='input-search'
+						className='input-search'
+						placeholder='Search for videos...'
+					/>
 
-				<div className='logout-wrapper'>
-					<UiwLogout
-						className='login-icon'
+					<FeSearch
+						className='search-icon'
+						stroke='black'
 						width='2rem'
 						height='1.5rem'
 					/>
 				</div>
+
+				{isUserObjEmpty ? (
+					<Link to='/login' className='login-link'>
+						<UiwLogout
+							width='2rem'
+							height='1.5rem '
+							className='login-icon'
+						/>
+					</Link>
+				) : (
+					<>
+						<button
+							onClick={() => logoutUser()}
+							className='logout-button'>
+							<UiwLogout
+								className='logout-icon'
+								width='2rem'
+								height='2rem'
+							/>
```

Comment:
suggestion: you can handle the css from class in css. It will be better in case you want to change it later.